src/algebra/lie/of_associative.lean

```diff
@@ -201,13 +201,31 @@ rfl
 
 variables {R L M}
 
-lemma lie_submodule.coe_map_to_endomorphism_le {N : lie_submodule R L M} {x : L} :
+namespace lie_submodule
+
+open lie_module
+
+variables {N : lie_submodule R L M} {x : L}
+
+lemma coe_map_to_endomorphism_le :
   (N : submodule R M).map (lie_module.to_endomorphism R L M x) ≤ N :=
 begin
   rintros n ⟨m, hm, rfl⟩,
   exact N.lie_mem hm,
 end
 
+variables (N x)
+
+lemma to_endomorphism_comp_subtype_mem (m : M) (hm : m ∈ N) :
```

Comment:
```suggestion
lemma to_endomorphism_comp_subtype_mem {m : M} (hm : m ∈ N) :
```

Reply:
I cannot make `m` implicit here without changing the explicitness of `x` in `(hf : ∀ x ∈ p, f x ∈ p)`, appearing in `linear_map.restrict.

Possibly this should be done but I'm reluctant to do it as part of these changes.

Reply:
OK, no problem.